server/manager/src/main/java/org/apache/accumulo/manager/tableOps/bulkVer2/BulkImportMove.java

```diff
@@ -99,6 +101,8 @@ private void moveFiles(FateId fateId, Path sourceDir, Path bulkDir, Manager mana
         "/" + bulkDir.getParent().getName() + "/" + bulkDir.getName(), fateId);
     AccumuloConfiguration aConf = manager.getConfiguration();
     int workerCount = aConf.getCount(Property.MANAGER_RENAME_THREADS);
+    ExecutorService workerPool = ThreadPools.getServerThreadPools()
```

Comment:
This still creates a thread pool per operation.  Need a single thread pool for the manager process.  This thread pool should be moved to the [Manager](https://github.com/apache/accumulo/blob/main/server/manager/src/main/java/org/apache/accumulo/manager/Manager.java).  Could add an instance variable to the manager class for the executor service, create the thread pool in the manager constructor, and add a a method to get the thread pool. With that done can do something like the following here.

```java
fs.bulkRename(oldToNewMap, manager.getRenameExecutor(), fateId);
```

Reply:
Hello,
I agree. 
I proposed singleton on the Manager level in the [issue](https://github.com/apache/accumulo/issues/5524#issuecomment-2920040513) discussion.